Tests/Twig/DataTableExtensionTest.php

```diff
@@ -6,14 +6,16 @@
 use CrossKnowledge\DataTableDundle\Tests\UsesContainerTrait;
 use Symfony\Component\DependencyInjection\Container;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use PHPUnit\Framework\TestCase;
+use Twig\Extension\AbstractExtension;
 
 require_once __DIR__.'/../UsesContainerTrait.php';
 
-class DataTableExtensionTest extends \PHPUnit_Framework_TestCase
+class DataTableExtensionTest extends TestCase
 {
     use UsesContainerTrait;
 
-    protected function getFunctionByName(\Twig_Extension $extension, $name)
+    protected function getFunctionByName(AbstractExtension $extension, $name)
     {
         foreach ($extension->getFunctions() as $function) {
             if ($function->getName()==$name) {
```

Comment:
If possible, could you please fix line 30 the same way you did in the Tests/DataTable/Renderer/TwigRendererTest.php file using SomeClass::class? 

(Sorry for the comment here, I can't write a comment in that line as you didn't change anything near there)

Reply:
Done.